src/components/collaboration/CollaborativeWhiteboard.tsx

```
@@ -0,0 +1,475 @@
+      ctx.lineJoin = 'round';
+
+      if (el.type === 'path' && el.points && el.points.length > 1) {
+        ctx.beginPath();
+        ctx.moveTo(el.points[0].x, el.points[0].y);
+        for (let i = 1; i < el.points.length; i++) {
+          ctx.lineTo(el.points[i].x, el.points[i].y);
+        }
+        ctx.stroke();
+      } else if (el.type === 'rectangle' && el.x !== undefined && el.y !== undefined && el.width && el.height) {
+        ctx.strokeRect(el.x, el.y, el.width, el.height);
+      } else if (el.type === 'circle' && el.x !== undefined && el.y !== undefined && el.width) {
```

ESLint

[error] Replace `'circle'` with `"circle"`

(prettier/prettier)